include/ck/tensor_operation/gpu/device/convolution_forward_specialization.hpp

```diff
@@ -5,12 +5,13 @@ namespace ck {
 namespace tensor_operation {
 namespace device {
 
-enum ConvolutionForwardSpecialization_t
+enum struct ConvolutionForwardSpecialization_t
 {
     Default,
     Filter1x1Pad0,
     Filter1x1Stride1Pad0,
     OddC,
+    UnDefined,
```

Comment:
What is `Undefined` specialization for? I can understand it being useful in C-style enum as a point where invalid values start. In enum structs invalid values would outright fail during compilation.

Reply:
good point

Reply:
fixed